lib/containers/container_images.pm

```diff
@@ -68,11 +68,11 @@ The main container runtimes do not need C<buildah> variable in general, unless
 you want to build the image with buildah but run it with $<runtime>
 
 =cut
-sub test_containered_app {
+sub build_and_run_image {
     my %args       = @_;
     my $runtime    = $args{runtime};
     my $buildah    = $args{buildah} // 0;
-    my $dockerfile = $args{dockerfile};
+    my $dockerfile = $args{dockerfile} // 0;
```

Comment:
this should the name of the Dockerfile template, right? See ` assert_script_run "curl -f -v " . data_url("containers/$file") . " > $dir/BuildTest/Dockerfile";` in [utils.pm](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/12760/files#diff-028c88dd89299f208a89ed70c6338724945d769d476e05f112201f7ad8bda770R211). Default `Dockerfile` otherwise whatever it is given.

Reply:
I am sorry, of course.